src/components/FormTitle/FormTitle.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./FormTitle.scss";
+
+export const FormTitle = (props) => {
+    return (
+        <div className="formTitle">
+            <div className="formTitle__text">{props.text}</div>
```

Comment:
You dont need nesting here, its redundant, having `formTitle__text` nested inside `formTitle` does not help you achieve anything you could not have done by putting all the styles just on `formTitle` and deleting the `formTitle__text`